scripts/rsl_run.py

```diff
@@ -0,0 +1,204 @@
+R"===(
```

Comment:
Potentially dumb question: what does "rsl" stand for in this filename?

Is this the same file contents as "run.py"?

Reply:
Raw String Literal, it is the same file as run.py, a feature of C++11. I recently realized that I can just include the run.py script and have the RSL identifiers just in the C++ file!

Reply:
I just found out that this doesn't work (https://stackoverflow.com/questions/52436628/how-to-include-text-file-as-string-at-compile-time-without-adding-c11-string-l) and we'd have to wait for `std::embed` aka c++2b slated for 2026.

I looked at the (CMake?)resource option, like with `builtin_kernels.cl`, but refrain for now for doing it that way, as that's all new for me and I want to get this merged asap :) For now I'll revert back to the rsl approach, which definitely isn't ideal since you have to update the script in two spots.